benchmarks/target/criterion/block_try_connect/base/benchmark.json

```diff
@@ -0,0 +1 @@
+{"group_id":"block_try_connect","function_id":null,"value_str":null,"throughput":null,"full_id":"block_try_connect","directory_name":"block_try_connect","title":"block_try_connect"}
```

Comment:
Should not commit the bench result.

Reply:
already remove target directory, thanks.